app/javascript/src/inscryb-mde.js

```diff
@@ -296,6 +301,44 @@ class InitialiseInscrybeMDE {
     }
   }
 
+  toggleVideoUploader() {
+    const button = this.mde.gui.toolbar.querySelector(".fa-video").closest("button")
+
+    button.classList.toggle("dropdown-open")
+
+    if (button.classList.contains("dropdown-open")) {
+      const dropdownElement = document.createElement("div")
+      dropdownElement.classList.add("editor-dropdown")
+
+      const textElement = document.createElement("small")
+      textElement.innerText = "Upload a video. Limited to mp4 filetype and 50mb filesize."
+
+      const randomId = Math.random().toString().substr(2, 8)
```

Comment:
Why use a `randomId` in the first place? I don't think there can be multiple of these dropdowns open, right?

It could just be a static, but very specific, ID. Something like "inscrybe-upload-video-file-input".

Reply:
Good question, I don't quite remember. This was copied from the image uploader

Reply:
Ah I remember now, it's because you can have the markdown editor open twice on the same page, for example 1 for the description and 1 for the revision description.